plugins/processors/ifname/ifname.go

```diff
@@ -301,7 +301,7 @@ func (d *IfName) makeTableNoMock(oid string) (*si.Table, error) {
 func (d *IfName) buildMap(gs snmp.GosnmpWrapper, tab *si.Table) (nameMap, error) {
 	var err error
 
-	rtab, err := tab.Build(gs, true, d.translator)
+	rtab, err := tab.Build(&gs, true, d.translator)
```

Comment:
I think I did a wrong fix here and it should actually have been `buildMap(gs *snmp.GosnmpWrapper, tab *si.Table)`?